### PR TITLE
Section 2.1. Processes

### PR DESCRIPTION
**Pull request by jurij-jukic**
Would appreciate a proof-read specifically on Wasm and Kinode stuff.

### CODE REVIEW

**Review by nick1udwig — APPROVED** [review]
A few comments. Otherwise LGTM; thanks!

File: src/process/processes.md [hunk 1]
```diff
@@ -176,7 +178,17 @@ There is more discussion of child processes [here](../cookbook/manage_child_proc
...
+Kinode processes are a further thing on top of components: they are components that follow a specific format in order to be run by Kinode OS.
 Pragmatically, processes can be compiled using the [`kit` tools](https://github.com/kinode-dao/kit).
-The long term goal of the Kinode runtime is to use [WASI](https://wasi.dev/) to provide a secure, sandboxed environment for processes to not only make use of the kernel features described in this document, but also to make full use of the entire WebAssembly ecosystem, including the ability to use sandboxed system calls provided by the host via WASI.
+
+Kinode can, using [WASI](https://wasi.dev/), provide a secure, sandboxed environment for Wasm components to make use of the kernel features described in this document. 
+Further, Kinode has a Virtual File System ([VFS](../files.md)) which processes can interact with, and therefore WASI could also expose access to filesystem for Wasm components directly.
+
+
+
+
```
Comment (nick1udwig) [inline src/process/processes.md:194]: Please remove these extra newlines

File: src/process/processes.md [hunk 2]
```diff
@@ -176,7 +178,17 @@ There is more discussion of child processes [here](../cookbook/manage_child_proc
...
 
+#### Wasm and Kinode
 It's briefly discussed here that processes are compiled to Wasm.
-The details of this are not covered in the Kinode Book, but can be found in the documentation for the [Kinode runtime](https://github.com/kinode-dao/kinode), which uses [Wasmtime](https://wasmtime.dev/), a WebAssembly runtime, to load, execute, and provide an interface for the subset of Wasm processes that are valid Kinode processes.
+The details of this are not covered in the Kinode Book, but can be found in the documentation for the [Kinode runtime](https://github.com/kinode-dao/kinode), which uses [Wasmtime](https://wasmtime.dev/), a WebAssembly runtime, to load, execute, and provide an interface for the subset of Wasm components that are valid Kinode processes.
+
+Wasm runs modules by default, or components, as described [here](https://component-model.bytecodealliance.org/design/why-component-model.html): components are just modules that follow some specific format.
+Kinode processes are a further thing on top of components: they are components that follow a specific format in order to be run by Kinode OS.
```
[the remark this answers is not in the document]
Comment (jurij-jukic) [reply src/process/processes.md:186]: do you want to replace only the line 186 here or all of them 183-186?

File: src/process/processes.md [hunk 3]
```diff
@@ -176,7 +178,17 @@ There is more discussion of child processes [here](../cookbook/manage_child_proc
...
 Pragmatically, processes can be compiled using the [`kit` tools](https://github.com/kinode-dao/kit).
-The long term goal of the Kinode runtime is to use [WASI](https://wasi.dev/) to provide a secure, sandboxed environment for processes to not only make use of the kernel features described in this document, but also to make full use of the entire WebAssembly ecosystem, including the ability to use sandboxed system calls provided by the host via WASI.
+
+Kinode can, using [WASI](https://wasi.dev/), provide a secure, sandboxed environment for Wasm components to make use of the kernel features described in this document. 
+Further, Kinode has a Virtual File System ([VFS](../files.md)) which processes can interact with, and therefore WASI could also expose access to filesystem for Wasm components directly.
```
[the remark this answers is not in the document]
Comment (jurij-jukic) [reply src/process/processes.md]: modified, also please confirm

File: src/process/processes.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nick1udwig) [reply src/process/processes.md]: Looks good :+1:

File: src/process/processes.md [hunk 4]
```diff
@@ -176,7 +178,17 @@ There is more discussion of child processes [here](../cookbook/manage_child_proc
...
+
+Wasm runs modules by default, or components, as described [here](https://component-model.bytecodealliance.org/design/why-component-model.html): components are just modules that follow some specific format.
+Kinode processes are a further thing on top of components: they are components that follow a specific format in order to be run by Kinode OS.
 Pragmatically, processes can be compiled using the [`kit` tools](https://github.com/kinode-dao/kit).
-The long term goal of the Kinode runtime is to use [WASI](https://wasi.dev/) to provide a secure, sandboxed environment for processes to not only make use of the kernel features described in this document, but also to make full use of the entire WebAssembly ecosystem, including the ability to use sandboxed system calls provided by the host via WASI.
+
+Kinode can, using [WASI](https://wasi.dev/), provide a secure, sandboxed environment for Wasm components to make use of the kernel features described in this document.
```
[the remark this answers is not in the document]
Comment (nick1udwig) [reply src/process/processes.md]: Looks good :+1:

File: src/process/processes.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (nick1udwig) [reply src/process/processes.md]: Looks good :+1:

**Comment by nick1udwig** [comment]
Looks good, thanks for the docs update :+1:

**Pull request merged**